tests/test_discretized_curves_space.py

```diff
@@ -0,0 +1,250 @@
+"""
+Unit tests for parameterized manifolds.
+"""
+
+import unittest
+
+import geomstats.backend as gs
+
+from geomstats.hypersphere import Hypersphere
+from geomstats.discretized_curves_space import DiscretizedCurvesSpace
+
+
+S2 = Hypersphere(dimension=2)
+R3 = S2.embedding_manifold
+
+INITIAL_POINT = [0, 0, 1]
+INITIAL_TANGENT_VEC_A = [1, 0, 0]
+INITIAL_TANGENT_VEC_B = [0, 1, 0]
+INITIAL_TANGENT_VEC_C = [-1, 0, 0]
+
+CURVE_A = S2.metric.geodesic(initial_point=INITIAL_POINT,
+                             initial_tangent_vec=INITIAL_TANGENT_VEC_A)
+CURVE_B = S2.metric.geodesic(initial_point=INITIAL_POINT,
+                             initial_tangent_vec=INITIAL_TANGENT_VEC_B)
+CURVE_C = S2.metric.geodesic(initial_point=INITIAL_POINT,
+                             initial_tangent_vec=INITIAL_TANGENT_VEC_C)
+
+N_SAMPLING_POINTS = 10
+SAMPLING_TIMES = gs.linspace(0, 1, N_SAMPLING_POINTS)
+POINT_A = CURVE_A(SAMPLING_TIMES)
+POINT_B = CURVE_B(SAMPLING_TIMES)
+POINT_C = CURVE_C(SAMPLING_TIMES)
+
+N_POINTS = 5
+TIMES = gs.linspace(0, 1, N_POINTS)
+ATOL = 1e-8
+
+
+class TestDiscretizedCurvesSpaceMethods(unittest.TestCase):
+    _multiprocess_can_split_ = True
+
+    def setUp(self):
```

Comment:
Actually, this should be `setUpClass(cls)`, otherwise it is run at the beginning and end of each test:
https://stackoverflow.com/questions/23667610/what-is-the-difference-between-setup-and-setupclass-in-python-unittest/23670844

We should correct it elsewhere in geomstats too :)